### DISCUSSION TITLE
Spin Components-fullScreen Prop

### DISCUSSION THREAD

**Discussion by Me5uT** [ant-design/ant-design#46983] [General]
I using "antd": "5.10.2" on my app. And I was using Spin Component for my loading page with fullScreen Prop. It can use with antd version "5.11.0" or higher on [documentation](https://ant.design/components/spin). But I have been using this feature for 2 months. So since yesterday I started getting errors about fullScreen prop.

**Comment by zombieJ** [comment]
`fullscreen` is added `5.11.0` which not exist in `5.10.0`. In most case it's other deps raise the antd version that make it work.

**Reply by Me5uT** [reply]
Until yesterday, I could use it with antd version "5.10.2". I getting errors now. I still using version "5.10.2".
